recipes/libtool/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+import os
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from contextlib import contextmanager
+
+
+class LibtoolConan(ConanFile):
+    name = "libtool"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/libtool/"
+    description = "GNU libtool is a generic library support script. "
+    topics = ("conan", "libtool", "configure", "library", "shared", "static")
+    license = ("GPL-2.0-or-later", "GPL-3.0-or-later")
+
+    settings = "os_build", "arch_build", "compiler"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def config_options(self):
+        # libtool provides a ltdl library, which is not packaged by this recipe
+        # To make the hooks happy, remove the c++ related settings
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    def requirements(self):
+        self.requires("automake/1.16.1")
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and "CONAN_BASH_PATH" not in os.environ \
+                and tools.os_info.detect_windows_subsystem() != "msys2":
+            self.build_requires("msys2/20190524")
+
+    @contextmanager
+    def _build_context(self):
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self.settings):
+                with tools.environment_append({"CC": "cl -nologo", "CXX": "cl -nologo",}):
+                    yield
+        else:
+            with tools.no_op():
```

Comment:
I guess it's not needed, as does nothing, right?

Reply:
I didn't about [tools.no_op](https://docs.conan.io/en/latest/reference/tools.html#tools-no-op), but it makes sense according the documentation

Reply:
Yup, the `no_op` is a nop here.